packages/react-query/src/useSuspenseInfiniteQuery.ts

```diff
@@ -27,7 +27,7 @@ export interface UseSuspenseInfiniteQueryOptions<
   > {}
 
 /**
- * This hook is wrapping useInfiniteQuery of @tanstack/react-query v4 with default suspense option.
+ * This hook is wrapping useInfiniteQuery of `@tanstack/react-query` v4 with default suspense option.
```

Comment:
I've made revisions for consistency.

Reply:
Thanks! So much better jsdoc 👍

### AS-IS
<img width="1111" alt="image" src="https://github.com/toss/suspensive/assets/61593290/86e1197e-1a54-460e-8cfa-b9ed70dd5c77">

### TO BE
<img width="1087" alt="image" src="https://github.com/toss/suspensive/assets/61593290/911523dc-307a-41a9-969a-052332544e3e">